docs/source/tracking.rst

```diff
@@ -270,13 +270,21 @@ store* specified as ``./path_to_store`` or ``file:/path_to_store``, or a SQL con
 for a *database-backed store*. For the latter, the argument must be a SQL connection string
 specified as ``db_type://<user_name>:<password>@<host>:<port>/<database_name>``. Supported
 database types are ``mysql``, ``mssql``, ``sqlite``, and ``postgresql``.
+For backwards compatibility, ``--file-store`` is an alias for --backend-store-uri
+
+.. important::
+
+    ``mlflow server`` will fail against a database-backed store with an out-of-date database schema.
+    To prevent this, upgrade your database schema to the latest supported version via
+    ``mlflow upgradedb [db_uri]``. Note that schema migrations can result in database downtime, may
+    take longer on larger databases, and are not guaranteed to be transactional. As such, consider
```

Comment:
Maybe we should word this a little more strongly:

```
always take a backup of your database prior to running ``mlflow upgradedb``
```

We should also indicate that the backup procedure depends on the user's underlying database (e.g., indicate that MLflow does not currently provide utilities for taking database backups).